tests/protocols/guidance_test.py

```diff
@@ -0,0 +1,178 @@
+from client.protocols import unlock
+from client.protocols import guidance
+from client.sources.common import models
+import mock
+import unittest
+import os
+
+
+class GuidanceProtocolTest(unittest.TestCase):
+	GUIDANCE_DIRECTORY = "demo/guidance_test/"
+	MISUCOUNT_FILE = "tests/misUcount.json"
+	TEST = "Test"
+	CASE_ID = TEST + ' > Suite 1 > Case 1 > Prompt 1'
+	UNIQUE_ID = "\nTest\n\n>>> 1 + 1 # OK will accept 'black' as right answer\ne74918d4310bb6cbc896676f20dc20de\n# locked\n>>> 2 + 2 # OK will accept 'black' as right answer\ne74918d4310bb6cbc896676f20dc20de\n# locked\n"
+	PROMPT = ">>> 1 + 1 # OK will accept 'black' as right answer"
```

Comment:
After we conanicalize the assessment IDs `UNIQUE_ID` and `PROMPT` will be need to be changed where the comments and hashed answered are removed.